packages/python/accern_data/util.py

```diff
@@ -85,32 +104,40 @@ def generate_file_response(
         df["harvested_at"] = pd.to_datetime(df["harvested_at"])
         df["published_at"] = pd.to_datetime(df["published_at"])
 
-        filtered = df[
+        valid_df = df[
             (df["published_at"] == date_dt) &
             (df["harvested_at"] > harvested_after_dt)
         ]
+        rows = []
+        for idx in range(valid_df.shape[0]):
+            if check_filters(valid_df.iloc[idx, :], filters):
```

Comment:
use a pandas filter function

Reply:
pandas.filter does not filter a dataframe on its contents.

Reply:
no not the function called filter -- pandas filters